js/Player.js

```diff
@@ -6,7 +6,11 @@ class Player {
   }
 
   increaseCurrentPlayerScore() {
+    if(wheel.currentSpinValue === 'BANKRUPT') {
+      this.score = 0;
+    } else {
     this.score += wheel.currentSpinValue;
+    }
   }
 
   decreaseCurrentPlayerScore() {
```

Comment:
Wheel should look at removing some of the methods on the player class that we are not using today.